languages/rust/oso/src/host/to_polar.rs

```diff
@@ -166,6 +167,12 @@ impl<T: ToPolar> ToPolar for HashMap<String, T> {
     }
 }
 
+impl ToPolar for PolarValue {
+    fn to_polar_value(self, _host: &mut Host) -> Value {
+        self.to_term().value().clone()
```

Comment:
Should `to_term()` instead be `to_value()`? It looks like this is the only usage & we are dropping the Term part.

Reply:
There's recursive calls that would have to be wrapped in term constructors, only the top level is unwrapped like this.